### PR TITLE
Add fr.romainvigier.MetadataCleaner

### PR DESCRIPTION
**Pull request by rmnvgr**
Hello,

I'd like to publish my new app [Metadata Cleaner](https://gitlab.com/rmnvgr/metadata-cleaner/) to Flathub!

Please let me know if there's anything wrong in the manifest.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build fr.romainvigier.MetadataCleaner

**Comment by rmnvgr** [comment]
bot, build fr.romainvigier.MetadataCleaner

File: fr.romainvigier.MetadataCleaner.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+  - name: metadata-cleaner
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.com/rmnvgr/metadata-cleaner.git
+        tag: v0.9.9-2
+        commit: 7ac462c43f826e9e04cad093be2a3d57ec8a5d98
+
+cleanup:
```
Comment (barthalion) [inline fr.romainvigier.MetadataCleaner.yaml:109]: FYI this is "traditionally" above `modules`.

Comment (rmnvgr) [reply fr.romainvigier.MetadataCleaner.yaml:109]: Thank you, I’ll update that.

**Pull request closed without merging**